content/page/contact.md

```diff
@@ -17,5 +17,5 @@ Pour des questions liés à l'organisation de meetups vous pouvez nous contacter
 aussi via :
 
 * notre groupe [meetup.com/Site-Reliability-Engineering-France](https://www.meetup.com/Site-Reliability-Engineering-France/)
-* par DM via notre compte Twitter [@SREFrance](https://twitter.com/SREFrance)
+* par DM via nos comptes [Mastodon @srefrance](https://piaille.fr/@srefrance), [Bluesky @srefrance](https://bsky.app/profile/srefrance.bsky.social) ou [X @SREFrance](https://x.com/SREFrance)
```

Comment:
While at here, what about having a clear listing (easier to share / browse):

```md
Pour suivre les actualités et/ou interagir avec la communauté:
* Telegram:
  * ReadWrite: https://t.me/+lH1dMlrK0q5hNTk8
  * ReadOnly: https://t.me/SREFranceNews
* Meetup: https://www.meetup.com/Site-Reliability-Engineering-France/
* Mastodon: https://piaille.fr/@srefrance
* BlueSky: https://bsky.app/profile/srefrance.bsky.social
* X: https://x.com/SREFrance
* Email: srefranceasso@gmail.com
```

Reply:
Yep, bonne idée pour simplifier le layout, je viens de commit un update.